js/controls.js

```diff
@@ -11,6 +11,12 @@
 		}
 	};
 
+	var trackMatomoEvent = function( eventCategory, eventAction, eventTitle ) {
+		if ( 'undefined' !== typeof _paq ) {
+			_paq.push([ 'trackEvent', eventCategory, eventAction, eventTitle ]);
```

Comment:
@KZeni Could we rename `trackGaEvent` to `trackAnalyticsEvent` and move `_paq.push()` to there instead? This would centralize the event tracking logic and we could move everything under the same toggle in the backend. Maybe rename it to "Analytics Tracking" and mention both GA and Matomo in the toggle description.

Reply:
I really did consider that, but I didn't want to intrude on the existing functionality. I'd be happy to go with that setup if you are!

Reply:
Yes, that would be great. Thanks!